content/en/security/application_security/troubleshooting.md

```diff
@@ -564,28 +564,41 @@ See the Application Security [set up documentation][11] to validate you you are
 
 Ensure the `DD_INSTRUMENTATION_TELEMETRY_ENABLED` environment variable (`DD_TRACE_TELEMETRY_ENABLED` for NodeJS) is set to `true`, or the corresponding system property for your language is enabled. For example in Java: `-Ddd.instrumentation.telemetry.enabled=true`
 
-## Disabling Threat Detection and Protection
+## Disabling threat detection and protection
 
 To disable ASM, remove the `DD_APPSEC_ENABLED=true` environment variable from your application configuration, and restart your service.
 
 If no `DD_APPSEC_ENABLED=true` environment variable is set for your service:
 * If it's a PHP service: explicitly set the environment variable to `DD_APPSEC_ENABLED=false`, and restart your service.
```

Comment:
In my understanding, PHP has 2 cases:
- local config (which is not an env variable but an ini file). 
- 1-click (a.k.a. remote config) which is what you describe below. 

I wonder if this alternative should be brought up as an edge case of the section above, vs in the no env variable set. 

@Anilm3 Can you tell us the best way for our users to know if ASM in PHP was configured locally or remotely, so we can clarify the docs a bit more? At the same time, can you double check this section of the enablement docs (https://docs.datadoghq.com/security/application_security/enabling/php?tab=dockercli#enabling-threat-detection) ? It's asking to set an env variable but I'm not sure it's relevant for PHP as it's handled by the install script, isn't it?

Reply:
> local config (which is not an env variable but an ini file).
> 1-click (a.k.a. remote config) which is what you describe below.

Currently the environment variable overrides both 1-click and local config. Howeve, if we want to document the local config case, we can mentioned where it's typically found, e.g. `/etc/php/<version>/(apache|fpm)/conf.d/98-ddtrace.ini` and the variable itself: `datadog.appsec.enabled = Off`

> Can you tell us the best way for our users to know if ASM in PHP was configured locally or remotely, so we can clarify the docs a bit more? 

The only way to know is through `phpinfo();` which has to be run within the application, not sure if documenting it helps, but I can gather some screenshots if needed.

> At the same time, can you double check this section of the enablement docs (https://docs.datadoghq.com/security/application_security/enabling/php?tab=dockercli#enabling-threat-detection) ? It's asking to set an env variable but I'm not sure it's relevant for PHP as it's handled by the install script, isn't it?

The documentation looks accurate as it includes this clarification:

> Enable the library in your code by restarting PHP-FPM or Apache. In a containerized environment, **if you previously installed the library without enabling ASM, you can optionally enable it after by setting the following environment variable:**